fs/layer/node.go

```diff
@@ -413,15 +459,18 @@ func (n *node) Listxattr(ctx context.Context, dest []byte) (uint32, syscall.Errn
 
 var _ = (fusefs.NodeReadlinker)((*node)(nil))
 
-func (n *node) Readlink(ctx context.Context) ([]byte, syscall.Errno) {
-	n.logOperation(ctx, "ReadLink")
+func (n *node) Readlink(ctx context.Context) (bt []byte, errno syscall.Errno) {
+	n.logOperation(ctx, fuseOpReadlink)
+	defer n.incOperationFailureMetric(ctx, fuseOpReadlink, errno)
```

Comment:
I added log and metrics for this one although it always return 0. I think it's worthwhile unless it's called very frequently. (same for `Statfs`)

Just in case later on if we change the implementation to returnning non-zero on error we don't need to add log/metric again.

Reply:
We shouldn't log an error metric if there's no error. It would only cause confusions.

Reply:
SGTM. I'll remove this one and below.